src/MultiField/MultiAssemblers.jl

```diff
@@ -107,8 +146,8 @@ end
 
 function assemble!(
   vec::Vector{E},
-  this::MultiSparseMatrixAssembler{E},
-  allmcv::Vararg{Tuple{<:MultiCellVector,<:CellNumber}}) where E
+  this::MultiSparseMatrixAssembler{E,M},
+  allmcv::Vararg{Tuple{<:MultiCellVector,<:CellNumber}}) where {E,M}
```

Comment:
I think this change is not needed (as before)

Reply:
solved in 53d940e